test/unit/anchore_engine/util/test_rpm.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from anchore_engine.util.rpm import compare_versions
+
+
+class TestRpmVersionHandling(unittest.TestCase):
+    """
+    Tests for version comparisons of the rpm version parser and compare code.
+
+    """
+
+    def test_version_comparision(self):
+        test_epoch = [
```

Comment:
I notice we don't have any tilde test cases. Since you're in here, please add a test for that case (e.g. 1.2.3-el7_5~snapshot1 < 1.2.3-3-el7_5). as well as for the dpkg